src/style.css

```diff
@@ -0,0 +1,35 @@
+
+body {
```

Comment:
Don't add redundant empty lines
```suggestion
body {
```